charts/agent/values.yaml

```diff
@@ -21,6 +21,16 @@ cluster:
     enabled: true
   metrics:
     enabled: true
+    pod:
+      enabled: false
+      action: keep
```

Comment:
what are the valid values here? and what does this do?

Reply:
enabled - true/false
action - keep / drop

Reply:
these flags end up having same effect - action is the way it was gated in grafana agent - I left it in but we should probably just delete it